packages/zipkin-transport-aws-sqs/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// @ts-ignore
```

Comment:
I am curious about this annotation, could you please ellaborate a bit on the reason?